src/controller/java/AndroidCallbacks.cpp

```diff
@@ -414,17 +420,29 @@ void ReportCallback::OnEventData(const app::EventHeader & aEventHeader, TLV::TLV
     // at the end.)
     TLV::TLVWriter writer;
     writer.Init(buffer.get(), bufferLen);
-    err = writer.CopyElement(TLV::AnonymousTag(), readerForJavaTLV);
+    TLV::TLVType outer;
+    err = writer.StartContainer(TLV::AnonymousTag(), TLV::kTLVType_Structure, outer);
```

Comment:
Same as above, we put everything under kTLVType_Structure, is this Structure temperately, and not in the final TLV payload

Reply:
context tag must be inside kTLVType_Structure

Reply:
See my comment above

Reply:
For event, we already put multiple type-value pairs under a struct, 
"0:STRUCT":{"0:UINT":1}

Wrap this struct in another struct seems extra

Reply:
for event, we don't need to put multiple events in structure, thanks